openedx/adg/lms/webinars/helpers.py

```diff
@@ -191,27 +191,40 @@ def save_scheduled_reminder_ids(mandrill_response, template_name, webinar_remind
         registration.save()
 
 
+def extract_emails_from_string(emails_string):
+    """
+    Get a list of emails from a string with comma separated emails
+
+    Arguments:
+        emails_string (str): string of comma separated emails
+
+    Returns:
+        list: List of comma separated emails
+    """
+    emails = [email.strip() for email in emails_string.split(',') if email.strip()]
+    return emails
+
+
 def validate_email_list(emails):
     """
-    Validate list of comma separated email addresses
+    Validate a list of email addresses
 
     Arguments:
-         emails (str): string of comma separated emails
+         emails (list): list of emails to validate
```

Comment:
This  will be string right? if not then how can we pass it to `extract_emails_from_string ` as a string? Implementation seems fine just update the docs here.